supernode/daemon/mgr/seed_task_mgr.go

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright The Dragonfly Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package mgr
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/dragonflyoss/Dragonfly/apis/types"
+)
+
+type SeedTaskInfo struct {
+	Load     int64 // load of peer
+	PeerInfo *types.PeerInfo
+	TaskInfo *types.TaskFetchInfo
+}
+
+type SeedRegisterResponse struct {
+	SeedTaskID string
+	AsSeed     bool
+}
+
+type SeedTaskMgr interface {
```

Comment:
comments a detailed information to describe what the interface does, also comments on struct `SeedTaskInfo`, `SeedRegisterResponse`.